plugins/woocommerce-admin/client/marketplace/components/extensions/extensions.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Is this necessary? I haven't noticed any linting errors for empty dependency comment blocks so not too sure what this is for. Also, this is going to disable that setting for the whole file, is it best doing it per line instead?

Reply:
I was getting the linting errors when running the linters locally. Added this to fix it. I have removed the `ProductList` component from this file for now.